manifests/init.pp

```diff
@@ -119,16 +117,4 @@
   }
 
   file { "${ldap::params::prefix}/${ldap::params::config}": content => template('ldap/ldap.conf.erb'), }
```

Comment:
I don't believe so, and I don't think this file is actually required for nslcd to function either.  Its only to be used as defaults for commands like ldapsearch and for pam_ldap.